book/03-git-branching/sections/basic-branching-and-merging.asc

```diff
@@ -139,38 +139,30 @@ index.html |    1 +
-In this case, Git does a simple three-way merge, using the two snapshots pointed to by the branch tips and the common ancestor of the two.
+Ito ay mukhang medyo iba sa `hotfix` na merge na ginawa mo kamakailan lamang. Sa kasong ito, ang kasaysayan ng iyong pag-develop ay humiwalay mula sa ilang mas matandang punto. Dahil ang commit sa branch kung saan nandoon ka ay hindi isang direktang ninuno ng branch na iyong mine-merge, ang Git ay dapat gumawa ng ilang trabaho. Sa kasong ito, ang Git ay gagawa ng isang simpleng three-way na merge, gamit ang dalawang snapshot na itinuro ng mga dulo ng branch at ang karaniwang ninuno ng dalawa.
 
-.Three snapshots used in a typical merge
-image::images/basic-merging-1.png[Three snapshots used in a typical merge.]
+.Tatlong snapshot na ginamit sa isang tipikal na merge
+image::images/basic-merging-1.png[Tatlong snapshot na ginamit sa isang tipikal na merge.]
 
-Instead of just moving the branch pointer forward, Git creates a new snapshot that results from this three-way merge and automatically creates a new commit that points to it.
-This is referred to as a merge commit, and is special in that it has more than one parent.
+Sa halip na ililipat lamang ang pointer ng branch nang pasulong, ang Git ay gagawa ng isang panibagong snapshot na nagreresulta mula sa three-way na merge na ito at awtomatikong bubuo ng isang panibagong commit na nakaturo nito. Ito ay tinutukoy bilang isang merge na commit, at ang espesyal nito ay mayroon itong higit pa sa isang magulang.
 
-.A merge commit
-image::images/basic-merging-2.png[A merge commit.]
+.Isang merge na commmit
+image::images/basic-merging-2.png[Isang merge na commmit.]
 
-It's worth pointing out that Git determines the best common ancestor to use for its merge base; this is different than older tools like CVS or Subversion (before version 1.5), where the developer doing the merge had to figure out the best merge base for themselves.
-This makes merging a heck of a lot easier in Git than in these other systems.
+Mahalagang pagtuunan ng pansin na tinutukoy ng Git ang pinakamabuti na karaniwang ninuno para gamitin sa merge base nito; ito ay iba sa mas matandang mga kasangkapan katulad ng CVS o Subversion (bago ang bersyon 1.5), kung saan ang developer na gumagawa ng merge ay kailangang malaman ang pinakamainam na merge base para sa sarili nila. Ginagawa nitong mas madali ang pag-merge sa Git kaysa sa mga ibang sistemang ito.
 
-Now that your work is merged in, you have no further need for the `iss53` branch.
-You can close the ticket in your ticket-tracking system, and delete the branch:
+Ngayong ang iyong trabaho ay naka-merge na, wala ka nang higit pang kailangan para sa `iss53` na branch. Maaari mo nang isara ang tiket sa iyong ticket-tracking na sistema, at burahin ang branch:
 
 [source,console]
 ----
 $ git branch -d iss53
 ----
 
 [[_basic_merge_conflicts]]
-==== Basic Merge Conflicts
+==== Batayan ng mga Salungatan sa Pag-merge
```

Comment:
I think this is much better :  Paunang Pagsamahin ang mga Salungatan

Reply:
How about "Mga Pangunahing Salungatan sa Pag-Merge"?

Reply:
@severinolorillajr i think it is much better one.

Reply:
I am a bit confused with this one. Merge conflicts here refers to the conflicts during the merging process. I think "Paunang mga Kasalungat sa Pag-merge" would make sense in this context somehow?

Reply:
I'll take shoganaii's on this one. Thank you.